### PR TITLE
boards: Rename opentitan to earlgrey-nexysvideo

### PR DESCRIPTION
**Pull request by gkelly**
### Pull Request Overview

This pull request renames `boards/opentitan` to `boards/earlgrey-nexysvideo`.

### Testing Strategy

This pull request was tested by running `make allboards` and `make prepush`.

### Documentation Updated

- [x] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [x] Ran `make prepush`.

### CODE REVIEW

**Comment by alistair23** [comment]
Looks like the CI is failing with formatting issues.

**Comment by gkelly** [comment]
> Looks like the CI is failing with formatting issues.

Strange, `make prepush` and `make ci-job-format` both run cleanly on my checkout.

**Review by alistair23 — APPROVED** [review]

**Review by alistair23 — APPROVED** [review]

**Comment by gkelly** [comment]
The netlify presubmit check shouldn't block submission, right?

**Comment by alistair23** [comment]
Nope, it's broken and no one can see why apparently: https://github.com/tock/tock/issues/2201

File: tools/board-runner/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ This can be used to perform Tock release testing on the OpenTitan board.
 This assumes that the OpenTitan serial console is available on the machines first serial port (`/dev/ttyUSB0` for Unix systems). The tests can be run from the top level of the Tock direcotry with the following command
 
 ```shell
-OPENTITAN_TREE=<opentitan_repo> LIBTOCK_C_TREE=<libtock_c_repo> TARGET=opentitan make board-release-test
+OPENTITAN_TREE=<opentitan_repo> LIBTOCK_C_TREE=<libtock_c_repo> TARGET=earlgret-nexysvideo make board-release-test
```
Comment (hudson-ayers) [inline tools/board-runner/README.md:14]:
```suggestion
OPENTITAN_TREE=<opentitan_repo> LIBTOCK_C_TREE=<libtock_c_repo> TARGET=earlgrey-nexysvideo make board-release-test
```

**Review by hudson-ayers — COMMENTED** [review]
We should probably change the name of the board in `boards/README` and `doc/wg/opentitan/README.md` as well, but I can't put suggestions in those files.

**Comment by gkelly** [comment]
> We should probably change the name of the board in `boards/README` and `doc/wg/opentitan/README.md` as well, but I can't put suggestions in those files.

I did change the `boards/README.md` file to call it out as "Earlgrey on Nexys Video".

File: tools/board-runner/README.md (hunk 1 above)
Comment (gkelly) [reply tools/board-runner/README.md]: Done.

**Comment by hudson-ayers** [comment]
> I did change the `boards/README.md` file to call it out as "Earlgrey on Nexys Video".

Sorry, I did a grep and saw an `[OpenTitan]` and didn't realize it was pointing to the out-of-tree board.

**Review by hudson-ayers — APPROVED** [review]

**Review by jrvanwhy — APPROVED** [review]

**Review by jon-flatley — APPROVED** [review]

**Pull request merged**